src/App.tsx

```diff
@@ -1,14 +1,41 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { handleFilteringTodos } from './utils/handleFilteringTodos';
+import { AllOptions, Todo } from './types';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isModal, setIsModal] = useState(false);
```

Comment:
You don't need this state, you can use `pressedTodo`(`selectedTodo`) state that we have below